composer.json

```diff
@@ -10,10 +10,10 @@
         "tijsverkoyen/css-to-inline-styles": "1.5.*",
         "matthiasmullie/minify": "~1.3",
         "matthiasmullie/scrapbook": "~1.0.3",
-        "symfony/symfony": "2.3.*",
-        "symfony/assetic-bundle": "~2.3",
-        "symfony/swiftmailer-bundle": "~2.3",
-        "symfony/monolog-bundle": "2.3.*",
+        "symfony/symfony": "^2.7",
```

Comment:
Ik ging eerst naar 2.7 upgraden (aangezien dit ook een LTS is en geen deprecation warnings geeft). Ik heb daarna geüpgrade naar 2.8 en zag dat die notices zeer eenvoudig te fixen waren. Uiteindelijk werkt al onze code op zowel 2.7 als 2.8 dus het lijkt me niet echt een probleem om zo te laten?